pkg/apis/core/types_shoot.go

```diff
@@ -75,7 +75,7 @@ type ShootSpec struct {
 	// Kubernetes contains the version and configuration settings of the control plane components.
 	Kubernetes Kubernetes
 	// Networking contains information about cluster networking such as CNI Plugin type, CIDRs, ...etc.
-	Networking Networking
+	Networking *Networking
```

Comment:
Networking is still needed for the service CIDR, right? Hence, no need to make it optional. Only the `Type` field in the `Networking` struct?

Reply:
Setting service CIDR is allowed, but not mandatory.

Reply:
Why is is not mandatory? Do you mean it's optional in the sense that it would be defaulted by Gardener if the user doesn't provide it? In the end, KAPI needs this CIDR, right?

Reply:
> In the end, KAPI needs this CIDR, right?

If service cidr is not provided, it defaults to `10.0.0.0/24` from kubernetes side.

Reply:
Ref https://github.com/gardener/gardener/pull/7828#discussion_r1176404003